### PR TITLE
fix bug 1454026 - redo breakpad scripts

### PR DESCRIPTION
**Pull request by willkg**

### CODE REVIEW

**Comment by willkg** [comment]
Up until this morning, I knew very very little about minidump-stackwalker. I decided to start some minidump-stackwalker documentation especially after seeing you use stackwalker as a diagnostic tool for processor issues.

Anyhow, as I understand it, we have four use cases here:

1. building the processor Docker image which the local dev environment and the server environments use
2. building breakpad
3. building stackwalker
4. building stackwalker from a locally-adjusted version of breakpad

Use cases 1, 2, and 3 all seem to work[1].

I want to know more about how you have 4 set up--which directories are where?

Let me know how this works for you and/or what changes we should make to make it work better.

[1] I have this weird issue on my machine where it pulls down `breakpad.tar.gz` from taskcluster and then fails when compiling stackwalker with missing symbol errors. I don't understand what I'm doing wrong. I don't need to support me, though.

**Comment by luser** [comment]
> I want to know more about how you have 4 set up--which directories are where?

I haven't done this in a while, because usually if I need to test something like this it's sufficient to just use the `minidump_stackwalk` tool from the Breakpad repository for testing.

File: minidump-stackwalk/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+files.
+
+
+Usage
+=====
+
+dumplookup
+----------
+
+FIXME(willkg): Document this.
```
[the remark this answers is not in the document]
Comment (willkg) [reply minidump-stackwalk/README.rst]: That'd be really great! Thank you!

**Comment by willkg** [comment]
Thank you for reviewing!

**Pull request merged**